sros2/api/__init__.py

```diff
@@ -340,6 +340,48 @@ def create_permission_file(path, name, domain_id, permissions_dict):
           </topics>
         </%s>
 """ % (tag, 'rt', topic_name, tag)
+        # TODO(mikaelarguedas) remove this hardcoded handling for default parameter topics
+        # TODO(mikaelarguedas) remove the need for empty partition (required for Connext at startup),
```

Comment:
fyi line length fixup in https://github.com/ros2/sros2/commit/0d4fc19c33adfde5ab9f50092bcf6131ea98e13e

Reply:
thanks!